server/odc-service/src/main/java/com/oceanbase/odc/service/permission/database/model/DatabasePermissionType.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.permission.database.model;
+
+import java.util.Locale;
+
+import org.springframework.context.i18n.LocaleContextHolder;
+
+import com.oceanbase.odc.common.i18n.Translatable;
+
+/**
+ * @author gaoda.xy
+ * @date 2024/1/3 14:04
+ */
+public enum DatabasePermissionType implements Translatable {
+
+    QUERY("query"),
```

Comment:
use UPPERCASE value straighlty

Reply:
But the `action` used in `iam_permission` is lowcase. This action is only a field for `DatabasePermissionType`, using lowcase is also reasonable.